mainnet/assets.json

```diff
@@ -817,6 +817,31 @@
         "discord": "https://discord.com/invite/58nxD6PncQ"
       },
       "extra": {}
+    },
+   {
+  "denom": "xrpl-placeholder-for-bwif-denom",
```

Comment:
`xrpl64430c4512-core1zhs909jp9yktml6qqx9f0ptcq2xnhhj99cja03j3lfcsp2pgm86studdrz`

Please set this as a denom